dracut/30ignition/ignition-files.service

```diff
@@ -5,30 +5,14 @@ DefaultDependencies=false
 # Ignition files stage starts after partitioning is completed.
 Requires=initrd-root-fs.target
 After=initrd-root-fs.target
+Before=initrd-parse-etc.service
```

Comment:
Would you mind leaving a comment above here to remind that Ignition can setup systemd config in the real rootfs, which are picked up by this service, thus the ordering?